yamllint/cli.py

```diff
@@ -49,6 +50,18 @@ def parsable(problem, filename):
                  'message': problem.message})
 
     @staticmethod
+    def json_output(problem, filename):
```

Comment:
Other formats don't have `_output`, you can simply name this function `json(problem, filename)` ;-)